src/visualization/types/Map/view.tsx

```diff
@@ -122,6 +124,7 @@ const GeoPlot: FC<Props> = ({result, properties}) => {
         colorField: '_value',
         colors: DEFAULT_THRESHOLDS_GEO_COLORS,
         isClustered: false,
+        tooltipColumns: tooltipColumns,
```

Comment:
So the code never reaches here, hence the tooltip columns is always `[]`. This is a safe fail guard against layers. You might want to do same thing as we are doing for colors in the line 133.